templates/blog/post_detail.html

```diff
@@ -0,0 +1,56 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+  <div style="margin-top: 20px">
+
+    <div class="post_detail_and_comments">
+      <h4>{{ post.title }}</h4>
+
+      <small class="text-muted">
+        <pre>{{ post.owner }} {{ post.created_time }}</pre>
+      </small>
+
+      <hr style="margin-top: 30px">
+      <p>{{ post.content }}</p>
+      <strong><p class="text-muted">Add new comment</p></strong>
+
+      <form method="post" action="{% url 'blog:comment-create' post.id %}" novalidate name="content">
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.